sources/PTYSession.m

```diff
@@ -2669,6 +2673,42 @@ - (void)setTransparency:(float)transparency
     [_textview setTransparency:transparency];
 }
 
+- (float)inactiveTransparency {
+    return [_textview inactiveTransparency];
+}
+
+- (void)setInactiveTransparency:(float)inactiveTransparency {
+    // Limit transparency because fully transparent windows can't be clicked on.
+    if (inactiveTransparency > 0.9) {
+        inactiveTransparency = 0.9;
+    }
+    [_textview setInactiveTransparency:inactiveTransparency];
+}
+
+- (BOOL)useInactiveTransparency {
+    return [_textview useInactiveTransparency];
+}
+
+- (void)setUseInactiveTransparency:(BOOL)useInactiveTransparency {
+    [_textview setUseInactiveTransparency:useInactiveTransparency];
+}
+
+- (BOOL)inactiveBlur {
+    return [iTermProfilePreferences boolForKey:KEY_INACTIVE_BLUR inProfile:_profile];
+}
+
+- (BOOL)blur {
+    return [iTermProfilePreferences boolForKey:KEY_BLUR inProfile:_profile];
+}
+
+- (float)inactiveBlurRadius {
+    return [iTermProfilePreferences floatForKey:KEY_INACTIVE_BLUR_RADIUS inProfile:_profile];
+}
+
+- (float)blurRadius {
```

Comment:
Both float and double are used here; I'm sure that's due to some ancient folly. Let's use double for all new code.